README.md

```diff
@@ -0,0 +1,60 @@
+<a href="https://www.twilio.com">
+  <img src="https://static0.twilio.com/marketing/bundles/marketing/img/logos/wordmark-red.svg" alt="Twilio" width="250" />
+</a>
+
+# Blog TFA Post - NodeJS/Express
+
+[![Build Status](https://travis-ci.org/TwilioDevEd/blog-tfa-node.svg?branch=master)](https://travis-ci.org/TwilioDevEd/blog-tfa-node)
+
+### Prerequisites
+
+1. [Node](http://nodejs.org/)
+1. [MongoDb](http://docs.mongodb.org/manual/installation/)
+1. A Twilio account with a verified [phone number](https://www.twilio.com/console/phone-numbers/incoming). (Get a
+   [free account](https://www.twilio.com/try-twilio?utm_campaign=tutorials&utm_medium=readme)
+   here.) If you are using a Twilio Trial Account, you can learn all about it
+   [here](https://www.twilio.com/help/faq/twilio-basics/how-does-twilios-free-trial-work).
+
+
+### Local Development
+
+1. First clone this repository and `cd` into it.
+
+   ```
+   $ git clone git@github.com:TwilioDevEd/blog-tfa-node.git
+   $ cd blog-tfa-node
+   ```
+
+1. Copy the sample configuration file and edit it to match your configuration.
+
+  ```bash
+  $ cp .env.example .env
+  ```
+
```

Comment:
If you use https://www.npmjs.com/package/dotenv there is no need to add an additional step like `source .env`.